tests/Vimeo/VimeoTest.php

```diff
@@ -0,0 +1,253 @@
+<?php
+namespace Vimeo\VimeoTest;
```

Comment:
We can just have this namespace be `Vimeo`. There won't be any collisions because the test case class is `VimeoTest`.